helm/slurm-cluster-storage/values.yaml

```diff
@@ -17,6 +17,11 @@ volume:
     name: "controller-spool"
     size: "30Gi"
     filestoreDeviceName: "controller-spool"
+  accounting:
+    enabled: true
```

Comment:
Why is accounting enabled by default during storage creation while it's by default disabled during cluster creation?